sql/core/src/test/resources/sql-tests/results/predicate-functions.sql.out

```diff
@@ -245,54 +245,54 @@ true
 -- !query
 select to_date('2009-07-30 04:17:52') <= to_date('2009-07-30 04:17:52')
 -- !query schema
-struct<(to_date('2009-07-30 04:17:52') <= to_date('2009-07-30 04:17:52')):boolean>
+struct<(to_date(2009-07-30 04:17:52) <= to_date(2009-07-30 04:17:52)):boolean>
```

Comment:
Sorry I happened to leave a comment late but why was this changed? Seems the previous one looks better.

Reply:
check this
```sql
+-- !query
+select date_format('2019-10-06', 'yyyy-MM-dd uuuu')
+-- !query schema
+struct<date_format(CAST(2019-10-06 AS TIMESTAMP), yyyy-MM-dd uuuu):string>
+-- !query output
+2019-10-06 Sunday
```

Reply:
Did we discuss about changing ^ case instead?

Reply:
Not yet. This PR just makes the `RuntimeReplaceable` -> `PrettyAttribute`. I agree with you that the `PrettyAttribute` is not beautiful in itself.